### DISCUSSION TITLE
Doesn't start kade engine

### DISCUSSION THREAD

**Discussion by AltronMaxX** [KadeArchive/Kade-Engine#2005] [Q&A]
I try to start kade engine without mods, but it starts for a second and then turns off.

**Comment by AltronMaxX** [comment]
No one can help me (

**Reply by FOXli12** [reply]
I have this problem too. Maybe there is something wrong with this latest version

**Reply by AltronMaxX** [reply]
I fixed that by starting kade engine from console

**Reply by FOXli12** [reply]
how to do it?
